ui/app/rif/pages/domainsDetailPage/domainDetailActive/domainDetailActive.js

```diff
@@ -0,0 +1,197 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCheckCircle, faArchive, faBolt, faChevronLeft, faPlusCircle } from '@fortawesome/free-solid-svg-icons'
+import { getIconForToken } from '../../../utils/utils'
+import { CustomButton, AddNewTokenNetworkAddress } from '../../../components'
+import rifActions from '../../../actions'
+import { cryptos } from '../../../constants'
+
+class DomainsDetailActiveScreen extends Component {
+	static propTypes = {
+		addNewNetwork: PropTypes.func.isRequired,
+		setAutoRenew: PropTypes.func.isRequired,
+		domainName: PropTypes.string.isRequired,
+		address: PropTypes.string.isRequired,
+		content: PropTypes.string.isRequired, 
+		expirationDate: PropTypes.string.isRequired,
+		autoRenew: PropTypes.bool.isRequired,
+		ownerAddress: PropTypes.string.isRequired,
+		isOwner: PropTypes.bool,
+		isLuminoNode: PropTypes.bool,
+		isRifStorage: PropTypes.bool,
+	}
+	constructor(props) {
+		super(props)
+		let resolvers = []
+		let networks = []
+		Object.assign(resolvers, props.resolvers)
```

Comment:
Same as previous comment. This can be done in one line and using const

Reply:
Fixed in next commit